### PR TITLE
add : home page

### PR DESCRIPTION
**Pull request by Varun-28**
added home page and made it responsive

home

### CODE REVIEW

File: css/base_styles/media.css [hunk 1]
```diff
@@ -7,11 +7,29 @@
...
+        grid-template-columns: 1fr 1fr;
+    }
+
+    .hero .hero-content{
+        text-align: left;
+        margin: 3rem 0 0 1rem;
+    }
+
+    .new-arrival-section .image-container{
+        grid-template-columns: repeat(4, 1fr) !important;
```
Comment (AnuragSahu11) [inline css/base_styles/media.css]: Use of !important is not recommended

**Review by AnuragSahu11 — COMMENTED** [review]
Other than !important tag everything looks good , proper use of naming conventions, you can merge

**Review by bhanuprakashj — COMMENTED** [review]
looks good to me other than !important usage which is already mentioned here

File: css/base_styles/media.css (hunk 1 above)
Comment (Varun-28) [reply css/base_styles/media.css]: done changes 👍

**Pull request merged**